tango_ros_common/tango_ros_native/src/tango_ros_node.cpp

```diff
@@ -351,7 +351,9 @@ TangoErrorType TangoRosNode::TangoSetupConfig() {
     return result;
   }
   const char* config_enable_depth = "config_enable_depth";
-  result = TangoConfig_setBool(tango_config_, config_enable_depth, true);
+  bool enable_depth;
+  node_handle_.param<bool>(ENABLE_DEPTH, enable_depth, true);
+  result = TangoConfig_setBool(tango_config_, config_enable_depth, enable_depth);
```

Comment:
No, of course these lines should stay to pick up changes when reconnecting to Tango. What I meant is that the ROS param should be initialized in the `onInit` function, like we do for the others.
Sorry for the confusion, I added this comment to the wrong line.